client/client/app/components/ngbBlastSearchPanel/ngbBlastWholeGenomeView/ngbBlastWholeGenomeView.scss

```diff
@@ -0,0 +1,17 @@
+md-dialog {
+  min-width: 800px;
+  min-height: 800px;
+  height: 800px;
+  background-color: #ffffff;
+  overflow-y: hidden;
+}
```

Comment:
This will apply styles to **all** dialogs of the app, which is obviously bad and not what we want:

**WAS**:
<img width="1434" alt="md-dialog" src="https://user-images.githubusercontent.com/5070496/113388703-647a1800-9397-11eb-97e8-ebaa4130654e.png">

**NOW**:
![md-dialog-broken](https://user-images.githubusercontent.com/5070496/113388750-765bbb00-9397-11eb-9ec7-6512a288bbfe.png)